content/300-accelerate/index.mdx

```diff
@@ -4,7 +4,7 @@ metaTitle: 'Prisma Accelerate'
 metaDescription: 'Prisma Accelerate is a global database cache with built-in connection pooling that helps improve database performance in Serverless and Edge applications.'
 sidebar_position: 0
 hide_table_of_contents: false
-pagination_next: 'accelerate/what-is-accelerate'
+# pagination_next: 'accelerate'
```

Comment:
meant to be commented?

Reply:
ah good catch, I commented it out to see if it was needed for some reason but don't think so, I guess it can be deleted entirely? 

```suggestion
```

Reply:
Unfortunately, these are needed. See the "next" button on the bottom of the page: https://gse-improvements.docs-51g.pages.dev/accelerate